<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>S1_Designation</class>
 <widget class="QMainWindow" name="S1_Designation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>S1_Designation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1151</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(128,128,128);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="designation">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>431</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_designation">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>391</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>select user type</string>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Admin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Receptionist</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Event Manager</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Staff Manager</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inventory Manager</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>381</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Select the Designation</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="login">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>381</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>User Id:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enter Sign-In Details :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>92</x>
         <y>200</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_input_username">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>60</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>Enter User Id</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_input_password">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>120</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>Enter Password</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="afterloginforADMIN">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>110</y>
        <width>441</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>140</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>this is after login  page for admin</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="afterloginforSTAFF">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>110</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>this is after login  page for staff</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="afterloginforINVENTORY">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>110</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>this is after login  page for inventory</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="afterloginforRECEPTIONIST">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>140</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>this is after login  page for receptionist</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="afterloginforEVENTMANAGER">
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>160</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>this is after login  page for event manager</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="optionsfortable">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>431</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>300</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>240</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Admin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Receptionist</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Event Manager</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Staff Manager</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inventory Manager</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>381</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Screen 2</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="changepassword"/>
    <widget class="QWidget" name="ADDformforEvent"/>
    <widget class="QWidget" name="ADDformforStaff"/>
    <widget class="QWidget" name="ADDformforInventory"/>
    <widget class="QWidget" name="ADDformforClient"/>
    <widget class="QWidget" name="ADDformforVenue"/>
    <widget class="QWidget" name="CreateUserform"/>
    <widget class="QWidget" name="PRINT"/>
    <widget class="QWidget" name="DELETE"/>
    <widget class="QWidget" name="UPDATE"/>
    <widget class="QWidget" name="VIEW"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
